patches/chrome-elevation_service-service_main.cc.patch

```diff
@@ -1,42 +1,13 @@
 diff --git a/chrome/elevation_service/service_main.cc b/chrome/elevation_service/service_main.cc
-index ef400534ef15cc8b7aeab32f0e71c664e7e5c0d3..83887db6257192a6f9d7c82c0d29af2cb522892f 100644
+index ef400534ef15cc8b7aeab32f0e71c664e7e5c0d3..db6d03d7be1eff2012a63fa01c83f3910512461c 100644
 --- a/chrome/elevation_service/service_main.cc
 +++ b/chrome/elevation_service/service_main.cc
 @@ -32,7 +32,7 @@ namespace {
  // Command line switch "--console" runs the service interactively for
  // debugging purposes.
  constexpr char kConsoleSwitchName[] = "console";
 -constexpr wchar_t kWindowsServiceName[] = L"ChromeElevationService";
-+std::wstring kWindowsServiceName;
++const wchar_t* kWindowsServiceName = GetWindowsServiceName();
```

Comment:
you cannot do that, because `GetWindowsServiceName` might be not initialized during startup (see `GetElevationServiceName() -> GetBaseAppName() -> InstallDetails::Get()`. You need to initialize it during `ServiceMain` construction as I proposed in a gist.

Reply:
ah right, I missed that `InstallDetails` would not be initialized yet when I suggested it

Reply:
Turns out - no patch needed! Removed in https://github.com/brave/brave-core/pull/20754/commits/be363eb58f75fe24dc5a5aab72240e123d6cac53